contribs/gmf/less/print.less

```diff
@@ -0,0 +1,88 @@
+.print-panel {
+  input[type=range] {
+    -webkit-appearance: none;
+    width: 100%;
+    padding-right: 0 !important;
+    padding-left: 0 !important;
+  }
+  input[type=range]:focus {
+    outline: none;
+  }
+  input[type=range]::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 5px;
+    cursor: pointer;
+    box-shadow: 1px 1px 1px rgba(0, 0, 0, 0), 0px 0px 1px rgba(13, 13, 13, 0);
+    background: rgba(148, 172, 154, 0.78);
+    border-radius: 1px;
+    border: 0.2px solid rgba(1, 1, 1, 0.2);
+  }
+  input[type=range]::-webkit-slider-thumb {
+    box-shadow: 0.9px 0.9px 1px rgba(0, 0, 49, 0.2), 0px 0px 0.9px rgba(0, 0, 75, 0.2);
+    border: 1px solid rgba(0, 0, 30, 0.2);
+    height: 20px;
+    width: 10px;
+    border-radius: 5px;
+    background: #ffffff;
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin-top: -7.7px;
+  }
+  input[type=range]:focus::-webkit-slider-runnable-track {
+    background: rgba(205, 217, 208, 0.78);
+  }
+  input[type=range]::-moz-range-track {
+    width: 100%;
+    height: 5px;
+    cursor: pointer;
+    box-shadow: 1px 1px 1px rgba(0, 0, 0, 0), 0px 0px 1px rgba(13, 13, 13, 0);
+    background: rgba(148, 172, 154, 0.78);
+    border-radius: 1px;
+    border: 0.2px solid rgba(1, 1, 1, 0.2);
+  }
+  input[type=range]::-moz-range-thumb {
+    box-shadow: 0.9px 0.9px 1px rgba(0, 0, 49, 0.2), 0px 0px 0.9px rgba(0, 0, 75, 0.2);
+    border: 1px solid rgba(0, 0, 30, 0.2);
+    height: 20px;
+    width: 10px;
+    border-radius: 5px;
+    background: #ffffff;
+    cursor: pointer;
+  }
+  input[type=range]::-ms-track {
+    width: 100%;
+    height: 5px;
+    cursor: pointer;
+    background: transparent;
+    border-color: transparent;
+    color: transparent;
+  }
+  input[type=range]::-ms-fill-lower {
+    background: rgba(95, 123, 102, 0.78);
+    border: 0.2px solid rgba(1, 1, 1, 0.2);
+    border-radius: 2px;
+    box-shadow: 1px 1px 1px rgba(0, 0, 0, 0), 0px 0px 1px rgba(13, 13, 13, 0);
+  }
+  input[type=range]::-ms-fill-upper {
+    background: rgba(148, 172, 154, 0.78);
+    border: 0.2px solid rgba(1, 1, 1, 0.2);
+    border-radius: 2px;
+    box-shadow: 1px 1px 1px rgba(0, 0, 0, 0), 0px 0px 1px rgba(13, 13, 13, 0);
+  }
+  input[type=range]::-ms-thumb {
+    box-shadow: 0.9px 0.9px 1px rgba(0, 0, 49, 0.2), 0px 0px 0.9px rgba(0, 0, 75, 0.2);
+    border: 1px solid rgba(0, 0, 30, 0.2);
+    height: 20px;
+    width: 10px;
+    border-radius: 5px;
+    background: #ffffff;
+    cursor: pointer;
+    height: 5px;
+  }
+  input[type=range]:focus::-ms-fill-lower {
+    background: rgba(148, 172, 154, 0.78);
+  }
+  input[type=range]:focus::-ms-fill-upper {
+    background: rgba(205, 217, 208, 0.78);
+  }
+}
```

Comment:
The input range styling should be made available not only for the print panel in my opinion but for all the application.